main.go

```diff
@@ -2,14 +2,46 @@ package main
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
 	"os"
 
 	"github.com/databricks/cli/cmd"
 	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/libs/telemetry"
+	"github.com/databricks/databricks-sdk-go/logger"
 )
 
 func main() {
 	ctx := context.Background()
+
+	// A new process is spawned for uploading telemetry data. We handle this separately
+	// from the rest of the CLI commands.
+	// This is done because [root.Execute] spawns a new process to run the
+	// "telemetry upload" command if there are logs to be uploaded. Having this outside
+	// of [root.Execute] ensures that the telemetry upload process is not spawned
+	// infinitely in a recursive manner.
+	if len(os.Args) == 3 && os.Args[1] == "telemetry" && os.Args[2] == "upload" {
+		// Suppress non-error logs from the SDK. This ensures that INFO / WARN logs
+		// from the SDK are not printed to stdout / stderr.
+		logger.DefaultLogger = &logger.SimpleLogger{
+			Level: logger.LevelError,
```

Comment:
Why not print warnings by default?

How to debug it if I want debug logs?

Reply:
Yeah, that's fine. I really wanted to suppress info logs since they otherwise leak into the acceptance test output.

> How to debug it if I want debug logs?

We can then flip this level manually. We can reevaluate this approach later if we find ourselves often having to debug telemetry.

Reply:
Why don't we inherit the log level from the main process? If I do 'bundle deploy --debug' and specify telemetry log file variable I'd expect telemetry logs to be debug level as well.

(In the future we'll add -v/-q to cli to manage log level more gradually.)